docs/content/tutorial/high-availability.md

```diff
@@ -391,6 +391,131 @@ spec:
+    
+### Example Spread Contraints
+
+To help illustrate how you might use this with your cluster, we can review examples for configuring spread constraints on our Instance and pgBackRest repo host Pods. For this example, assume we have a three node Kubernetes cluster where the first node is labeled with `zone=zone1`, the second node is labeled with `zone=zone2` and the final node is labeled `zone=zone3`. The label key `zone` will function as our `topologyKey`. Note all three nodes in our examples will be schedulable, so a Pod could live on any of the three Nodes.
+
+#### Instance Pod Spread Constraints
+
+To begin, we can set our topology spread contraints on our cluster Instance Pods. Given this configuration
+
+```
+  instances:
+    - name: instance1
+      replicas: 5
+      topologySpreadConstraints: 
+        - maxSkew: 1
+          topologyKey: zone
+          whenUnsatisfiable: DoNotSchedule
+          labelSelector: 
+            matchLabels:
+              postgres-operator.crunchydata.com/instance-set: instance1
+```
+
+we will expect 5 Instance pods to be created. Each of these Pods will have the standard `postgres-operator.crunchydata.com/instance-set: instance1` Label set, so each Pod will be properly counted when determining the `maxSkew`. Since we have 3 nodes with a `maxSkew` of 1 and we've set `whenUnsatisfiable` to `DoNotSchedule`, we should see 2 Pods on 2 of the nodes and 1 Pod on the remaining Node, thus ensuring our Pods are distributed as evenly as possible.
+
+#### pgBackRest Repo Pod Spread Constraints
+
+We can also set topology spread constraints on our cluster's pgBackRest repo host pod. While we normally will only have a single pod per cluster, we could use a more generic label to add a preference that repo host Pods from different clusters are distributed among our Nodes. For example, by setting our `matchLabel` value to `postgres-operator.crunchydata.com/pgbackrest: ""` and our `whenUnsatisfiable` value to `ScheduleAnyway`, we will allow our repo host Pods to be scheduled no matter what Nodes may be available, but attempt to minimize skew as much as possible.
+
+```
+      repoHost:
+        topologySpreadConstraints: 
+        - maxSkew: 1
+          topologyKey: zone
+          whenUnsatisfiable: ScheduleAnyway
+          labelSelector: 
+            matchLabels:
+              postgres-operator.crunchydata.com/pgbackrest: ""
+```
+
+#### Putting it All Together
+
+Now that each of our Pods has our desired Topology Spread Constraints defined, let's put together a complete cluster definition:
+
+```
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: hippo
+spec:
+  image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres-ha:centos8-13.4-0
+  postgresVersion: 13
+  patroni:
+    dynamicConfiguration:
+      postgresql:
+        parameters:
+          shared_preload_libraries: pgaudit.so
```

Comment:
⛏️ Some of these fields aren't related to the topic of spread constraints. Can we ditch dynamic config, WAL volume, and images?

Reply:
Agree re: dynamic config and WAL volume. I'd like to leave images in place to stay consistent with the rest of the high-availability section.